README.rst

```diff
@@ -35,7 +35,7 @@ contains the package `django-two-factor-auth`_, but that application is not a
 dependency for this package. Also have a look at the bundled example templates
 and views to see how you can integrate the application into your project.
 
-Compatible with Django 2.2, 3.0 and 3.1 on Python 3.6, 3.7, 3.8 and 3.9.
+Compatible with Django 2.2, 3.0, 3.1, 3.2 and 4.0 on Python 3.6, 3.7, 3.8, 3.9 and 3.10.
```

Comment:
If we're not testing against Django 2.2, 3.0, and 3.1,  we probably shouldn't be claiming compatibility with them. We should probably fully remove anything indicating support prior to 3.2 with this PR.

Reply:
When I opened this PR Django 2.2 was still supported. As this no longer is the case I agree on that it would be better to just remove all mentions of Django versions before 3.2.

The tox tests do actually also run using 3.6 and Django 2.2. I did however in my latest addition switch to only running the GitHub action tests using more recent versions (because running them against all versions would have required using several jobs due to version incompatibilities).

I'll update the tests and readme.